app/lib/features/space/widgets/space_header_profile.dart

```diff
@@ -42,8 +42,20 @@ class SpaceHeaderProfile extends ConsumerWidget {
                         ),
                       ]
                     : [],
-                tooltip: TooltipStyle.None,
-                badgeSize: 40,
+                onAvatarTap: () => context.pushNamed(
+                  Routes.space.name,
+                  pathParameters: {
+                    'spaceId': spaceId,
+                  },
+                ),
+                onParentBadgeTap: () => context.pushNamed(
+                  Routes.space.name,
+                  pathParameters: {
+                    'spaceId':
+                        canonicalParent.valueOrNull!.space.getRoomIdStr(),
+                  },
+                ),
```

Comment:
Are we using `onParentBadgeTap` in space header only? I mean I don't see that in other pages like RoomAvatar or widget where we are showing spaceBadge.

Reply:
Do we need to give interaction for parent badge everywhere? I need clarification on this. For chat list, it doesn't make sense and would be very weird and confusing if someone taps on room and takes to parent space.